doc/source/whatsnew/v1.5.0.rst

```diff
@@ -295,7 +295,7 @@ Conversion
 
 Strings
 ^^^^^^^
--
+- Bug in :meth:`str.startswith` and :meth:`str.endswith` when using other series as parameters. (:issue:`3485`)
```

Comment:
Good to mention that a `TypeError` now raises when not passing a string as the param.